index.html

```diff
@@ -215,231 +215,186 @@ <h3>Motivation for a Profile</h3>
+      which is guaranteed to be able to communicate with any
+      <a href="#dfn-thing">Web Thing</a> [[wot-architecture]] out of the box.
+    </p>
+    <p>
+      This specification is designed to complement the Thing Description
+      [[wot-thing-description]] specification, by defining a
+      <a href="#profiling-mechanism">Profiling Mechanism</a> and a set of
+      <a href="#dfn-profile">Profiles</a> which enable
+      <a href="#out-of-the-box-interoperability">out-of-the-box
+      interoperability</a> between <a href="#dfn-thing">Web Things</a> and their
+      <a href="#dfn-consumer">Consumers</a>.
+    </p>
+    <p>
+      Profiles are designed specifically for new ("greenfield") implementations
+      where developers have the freedom to conform to a prescriptive protocol
+      binding and set of common constraints, in order to benefit from
+      out-of-the-box interoperability.
+    </p>
+    <p class="note">
+      Of the Profiles defined in this specification, only the
+      <a href="#http-basic-profile">HTTP Basic Profile</a> is currently
+      normative. It is planned that future versions of this specification will
+      normatively define the <a href=#sec-http-sse-profile>HTTP SSE Profile</a>
+      and <a href="#sec-http-webhook-profile">HTTP Webhook Profile</a>, but
+      in this version of the specification they are only informative.
+    </p>
+    <p>
+      Future versions of this specification, or extension specifications, may
+      define additional Profiles.
+    </p>
+  </section>
+  <section id="profile-example">
+    <h2>Profile Example</h2>
+    <p>
+    <a href="#simple-profile-td-sample">Example 1</a> shows a <a>Thing Description</a>
+    instance that follows the <a>HTTP Basic Profile</a> as described in this
+    sepecification.
+    </p>
+  <aside class="example" id="simple-profile-td-sample" title="Thing Description follows the <a>HTTP Basic Profile</a>">
+      <pre>{
+      "@context": "https://www.w3.org/2022/wot/td/v1.1",
+      "id": "urn:uuid:0804d572-cce8-422a-bb7c-4212fcd56f06",
+      "profile": "https://www.w3.org/2022/wot/profile/http-basic/v1",
+      "title": "MyHTTPBasicProfileLampThing",
+      "base": "https://mylamp.example.com",
+      "securityDefinitions": {
+          "basic_sc": {"scheme": "basic", "in": "header"}
+      },
+      "security": "basic_sc",
+      "properties": {
+          "status": {
+              "title": "Status",
+              "type": "string",
+              "forms": [{"href": "/status"}]
+          }
+      },
+      "actions": {
+          "toggle": {
+              "title": "Toggle",
+              "synchronous": true,
+              "forms": [{"href": "/toggle"}]
+          }
+      }
+  }</pre>
+  </aside>
+  <p>
+  This <a>Thing Description</a> uses the <code>profile</code> term to signalize that the associated <a>Thing</a>
+  supports the <a>HTTP Basic Profile</a>. As a security scheme, this example enforces authentication before access
```

Comment:
"No matching definition" for "HTTP Basic Profile", need to add `href="#http-basic-profile`.